src/Application/Contracts/IUserService.cs

```diff
@@ -38,5 +42,10 @@ public interface IUserService
         Task<Response<string>> ValidateEmailOtpAsync(string requestMfaDetails, string code);
         Task<string> GetMaskedMobile(string requestMfaDetails);
         Task<Response<string>> ReissueMfaSessionAsync(string requestMfaDetails);
+        Response<string> HandleMfaException(Exception ex, string errorType);
+        AdminRespondToAuthChallengeRequest CreateAuthChallengeRequest(string challengeName, string sessionId,
+            string username, string code, string codeKey);
+        MfaLoginDetails DeserializeMfaLoginDetails(string mfaDetails);
```

Comment:
I don't think this makes sense on this interface.
The implementation is very simple so you could duplicate the code if you wanted to or add it to `MfaLoginDetails`.